multiplayer/examples/sudoku/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "pinpoint",
+  "version": "0.1.0",
+  "private": true,
+  "homepage": ".",
+  "dependencies": {
+    "@lottiefiles/react-lottie-player": "^3.5.3",
+    "@rollup/plugin-commonjs": "^24.1.0",
```

Comment:
Not sure how much we care about it, but many of these are devDependencies

Reply:
I'm ok either way. `devDependencies` don't have much sense in leaf projects since they never become packages